qa/acceptance-tests/src/test/java/io/camunda/it/csrf/CsrfTokenIT.java

```diff
@@ -39,6 +37,8 @@
 @DisabledIfSystemProperty(named = "test.integration.camunda.database.type", matches = "AWS_OS")
 public class CsrfTokenIT {
 
+  // TODO: assert that there are exactly two cookies
```

Comment:
Consider adding a ticket (or any other follow up item) to `TODO`

Reply:
Ah, I added the test in fact but I didn't remove the todo.